rfc3164.go

```diff
@@ -43,6 +43,56 @@ const (
 // Facility represents a syslog facility code
 type Facility int
 
+//FacilityTextToFacility accepts a string representation of a syslog
+// facility and returns a captainslog.Facility
+func FacilityTextToFacility(facilityText string) (Facility, error) {
+	var err error
```

Comment:
@bryanl - explored using a map - the problem is that accessing a key that didn't exist would I believe return the equivalent of Facility(0), which is the Kern facility.